packages/exporter-otlp-http/src/index.ts

```diff
@@ -77,6 +79,14 @@ export function init({
 }: InitOptions) {
   amLogger.info(`Exporter will push to the OTLP/HTTP endpoint at ${url}`);
 
+  const urlObj = new URL(url);
+
+  if (urlObj.pathname !== "/v1/metrics") {
```

Comment:
Think we should check if it ends with this path instead if it equals, as it is possible to have a prefix, but still end with `/v1/metrics`. I think Prometheus has it under a prefix, but still ending with `/v1/metrics`.